Editor/Scripts/Utilities/MeshsyncAnalytics.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+using UnityEditor;
+
+using UnityEngine;
+
+namespace Unity.MeshSync.Editor.Analytics {
+
+    internal static class MeshSyncAnalyticsFactory {
+
+        public static IMeshSyncAnalytics CreateAnalytics() {
+            return new MeshSyncAnalytics();
+        }
+    }
+
+    internal interface IMeshSyncAnalytics {
+
+        /// <summary>
+        /// Send an event that a user installed and configured a DCC tool for use with MeshSync
+        /// </summary>
+        /// <param name="dccTool">Short designation for the DCC which was installed and configured</param>
+        void UserInstalledPlugin(string dccTool);
+
+        void UserSyncedData(MeshSyncAnalyticsData data);
+    }
+
+    internal sealed class MeshSyncAnalytics : IMeshSyncAnalytics {
+        private const string VENDORKEY = "unity.meshsync";
+
+        private const string DCCCONFIGURE_INSTALLEDEVENTNAME = "dccConfigure";
+        private const int DCCCONFIGURE_VERSION = 4;
+
+        private const string MESHSYNC_SYNC_INSTALLEDEVENTNAME = "meshSync_Sync";
+        private const int MESHSYNC_SYNC_VERSION = 2;
```

Comment:
Are these versions supposed to correspond to the release version or some sort of event versioning?

Reply:
Answered in Sean's review